kube-ps1.sh

```diff
@@ -334,8 +334,21 @@ kubeoff() {
   KUBE_PS1_ENABLED=off
 }
 
+# make sure we have a configuration available
+kubeconfig_avail() {
+  if [[ -f "${HOME}/.kube/config" ]] || [[ -n "${KUBECONFIG}" ]]; then
```

Comment:
Sorry, @jonmosco , it doesn't work, and I don't mean to be harsh, but I don't understand how you can think this would work. If `$KUBECONFG` is set to `~/foo` and `kube-ps1` finds that file and sets up the prompt accordingly, and then I delete `~/foo`, nothing in this conditional will change.


What I think you should do is update https://github.com/jonmosco/kube-ps1/blob/c0336594b9854e4e27f00c892bf87ccdddfd2891/kube-ps1.sh#L227-L234

so that it detects that nothing was found. Something like

```bash
local conf 
local conf_found
 for conf in $(_kube_ps1_split : "${KUBECONFIG:-${HOME}/.kube/config}"); do 
   [[ -r "${conf}" ]] || continue 
  conf_found="${conf}"
   if _kube_ps1_file_newer_than "${conf}" "${KUBE_PS1_LAST_TIME}"; then 
     _kube_ps1_get_context_ns 
     return 
   fi 
 done 
 [[ -z "${conf_found}" ]] && KUBE_PS1_CONTEXT=""
```

Reply:
I was attempting to not display the prompt if the kubeconfig was not available is all.  Harsh is fine, I was attempting a quick fix.